src/resmom_win/start_exec.c

```diff
@@ -195,6 +196,94 @@ static	void catchinter(int);
 	fd = hold; \
 }
 
+/**
+ * @brief
+ *	Update job's nodes resources information based on the
+ *	hook set values to exec_vnode, exec_host/exec_host2.
+ *
+ * @param[in,out]	pjob	 - job being operated on.
+ * @param[in]		old_exec_vnode - job's previous exec_vnode
+ *					value before the hook update.
+ * @retval 0	- for success
+ * @retval 1	- for failure
+ */
+static int
+send_update_job(job *pjob, char *old_exec_vnode)
+{
+	int	exec_vnode_hookset;
+	int	schedselect_hookset;
+	int	exec_host_hookset;
+	int	exec_host2_hookset;
+	char	*new_exec_vnode;
+	char	*new_exec_host;
+
+	if (pjob == NULL)
+		return (1);
+
+	exec_vnode_hookset = pjob->ji_wattr[JOB_ATR_exec_vnode].at_flags & ATR_VFLAG_HOOK;
+	schedselect_hookset = pjob->ji_wattr[JOB_ATR_SchedSelect].at_flags & ATR_VFLAG_HOOK;
+	exec_host_hookset = pjob->ji_wattr[JOB_ATR_exec_host].at_flags & ATR_VFLAG_HOOK;
+	exec_host2_hookset = pjob->ji_wattr[JOB_ATR_exec_host2].at_flags & ATR_VFLAG_HOOK;
+	if (!exec_vnode_hookset || !schedselect_hookset ||
+	   (!exec_host_hookset && !exec_host2_hookset)) {
+		return (1);
+	}
+
+	snprintf(log_buffer, sizeof(log_buffer), "pruned from exec_vnode=%s",
+	     	pjob->ji_wattr[(int)JOB_ATR_exec_vnode].at_val.at_str);
+	log_event(PBSEVENT_JOB, PBS_EVENTCLASS_JOB, LOG_INFO,
+			pjob->ji_qs.ji_jobid, log_buffer);
+
+	snprintf(log_buffer, sizeof(log_buffer), "pruned from exec_vnode=%s",
+	     	old_exec_vnode);
```

Comment:
Both these log messages say'prune from exec_vnode'.  Also here again, you're printing the exec_vnode twice at a log level on by default.  This will be messy. (you actually do this 4 times)